.jenkins.groovy

```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
 
+def supportedBranch = env.BRANCH_NAME ==~ /(1.7.x|1.8.x|main)/
```

Comment:
"supported"? Like in "we give support"? Maybe just change to "deployable branch" or "maintained" or something like this. It should be clear that this *only* has to do with snapshot deployments.

Reply:
Good call, I wasn't happy with this name either, `deployableBranch` sounds good to me!

Reply:
@bmarwell back to you when you get a chance